sentences/en/_common.yaml

```diff
@@ -361,28 +361,36 @@ lists:
     wildcard: true
 
 expansion_rules:
-  name: "[the] {name}"
-  area: "[the] {area}"
-  floor: "[the] {floor}[ floor]"
-  what_is: "(what's|whats|what is)"
+  name: "[the ]{name}"
+  area: "[the ]{area}"
+  floor: "[the ]{floor}[ floor]"
+  area_floor: "(<area>|<floor>)"
+  in_area_floor: "[<in> ]<area_floor>"
+  what_is: "(what's|whats|what is|tell me)[ the]"
   where_is: "(where's|wheres|where is)"
+  which: "(which|what)[ of the]"
+  is: "(is|are)[ (there|the)]"
+  are: "<is>"
+  any: "[<are> ](any|some)[ of the]"
+  how_many: "how many[ of the]"
   brightness: "{brightness}[[ ]%| percent]"
-  light: "[the] (light|lights|lighting)"
+  light: "[the ](light|lights|lighting|lamp|lamps)"
   turn: "(turn|switch|change)"
   temp: "(temp|temperature)"
   temperature: "{temperature}[([ ]°[ ][{temperature_unit}])|( degrees[ {temperature_unit}])]"
-  open: "(open|raise)"
-  close: "(close|shut|lower)"
+  open: "(open|raise|lift)[ up]"
+  close: "(close|shut|lower)[ (up|down)]"
   set: "(set|make|change|turn)"
-  numeric_value_set: "(set|change|turn|increase|decrease|make)"
+  numeric_value_set: "(set|change|turn[ (up|down)]|increase|decrease|make)"
   in: "(in|on|at|of)"
   position: "{position}[[ ]%| percent]"
   volume: "{volume:volume_level}[[ ]%| percent]"
+  state: "[(present|current) ](state|status)"
 
   # Context awareness expansion rules
-  all: "(all|all of|every|every single|each|each and every)"
+  all: "[<are> ](all[[ of] the]|every[ single]|each[ and every])"
```

Comment:
Why `[<are> ]`? Looking where this expansion rule was used prior to this PR, it makes no sense

Reply:
This is for the `all` response type. The question being "Are all the shades open?" Or "Are all the shades open?". The sentences could use `[<are> ]<all>`, but I was hoping to make this easier from a maintainer perspective so that `all` responses are basically `<all>` and then the thing.

That said, the `all` responses could probably be refactored to be extremely generic since they can more or less be generalized as: `<are> <all> {{domain}_classes:device_class} <is> {{domain_states}:state}[ <in_area_floor>]`, if we had a way to expand to all domains or even just generate for all domains at build/deploy. That would make maintenance much simpler and negate the benefits of making it easier to maintain at a per domain level.

Reply:
Don't get me wrong, it would be great if the expansion rule was called `<are_all>` or something, but this way it is just confusing. And `<all>` is used in sentences such as [this](https://github.com/home-assistant/intents/blob/1242f1209b364af6a99179467ab24b488a0a8879/sentences/en/light_HassTurnOff.yaml#L14).

Reply:
Ah. That's a good point. `<are_all>` makes sense to me.

Have any idea where to start to implement the latter idea so we don't even have to do this for each domain? Is that feasible without an intent processing change?

Reply:
IT is. Here's how I did it in Romanian:
1. create [an expansion rule](https://github.com/home-assistant/intents/blob/1242f1209b364af6a99179467ab24b488a0a8879/sentences/ro/_common.yaml#L628) that uses other undefined expansion rules
2. add the expansion rule as a sentence template in [multiple](https://github.com/home-assistant/intents/blob/1242f1209b364af6a99179467ab24b488a0a8879/sentences/ro/binary_sensor_HassGetState.yaml#L782) [sentences](https://github.com/home-assistant/intents/blob/1242f1209b364af6a99179467ab24b488a0a8879/sentences/ro/binary_sensor_HassGetState.yaml#L998) for [multiple](https://github.com/home-assistant/intents/blob/1242f1209b364af6a99179467ab24b488a0a8879/sentences/ro/binary_sensor_HassGetState.yaml#L1165) use cases
3. in each sentence, [populate](https://github.com/home-assistant/intents/blob/1242f1209b364af6a99179467ab24b488a0a8879/sentences/ro/binary_sensor_HassGetState.yaml#L786) the missing pieces using [local expansion rules](https://developers.home-assistant.io/docs/voice/intent-recognition/template-sentence-syntax#local-expansion-rules)